GalnetMonitor/GalnetFeedItemNormalizer.cs

```diff
@@ -46,17 +46,17 @@ public virtual FeedItem Normalize(SyndicationFeed feed, SyndicationItem item)
 
             return new ExtendedFeedItem
             {
-                Id = string.IsNullOrEmpty(item.Id) ? null : item.Id.Trim(),
-                Title = item.Title == null ? null : Normalize(item.Title.Text),
-                Content = item.Content == null ? null : Normalize(((TextSyndicationContent)item.Content).Text),
-                Summary = item.Summary == null ? null : Normalize(item.Summary.Text),
+                Id = string.IsNullOrEmpty(item.Id) ? null : item.Id.Split(' ')[0].Trim(),
```

Comment:
Splitting is necessary because the feed item title is appended here for some reason.